base/v0_5_exp/translate.go

```diff
@@ -217,6 +218,48 @@ func translateLink(from Link, options common.TranslateOptions) (to types.Link, t
 	return
 }
 
+func translateSSHAuthorizedKeysLocal(from []SSHAuthorizedKeyFile, options common.TranslateOptions) (to []types.SSHAuthorizedKey, tm translate.TranslationSet, r report.Report) {
```

Comment:
Does this work if both inline and local keys are specified?  I wouldn't be surprised if it doesn't, and the tests aren't checking for that.  I suspect the translator should go on the parent struct instead.